CHANGES.rst

```diff
@@ -11,7 +11,7 @@ Bugfix:
  -
 
 API Change:
- -
+ - We no longer add the history_visibility parameter when creating a room. Now we use the method setHistoryVisibility.
```

Comment:
Write:
` - parameter historyVisibility removed from MxSession.createRoom(). It had no effect.`

Reply:
This phrase is supposed to replace the phrase you wrote :)